jiant/utils/retokenize.py

```diff
@@ -95,35 +95,6 @@ def _mat_from_spans_sparse(spans: Sequence[Tuple[int, int]], n_chars: int) -> Ma
     return sparse.csr_matrix((data, (ridxs, cidxs)), shape=(len(spans), n_chars))
```

Comment:
I don't really know the logic in this file. You'll need somebody else to take a look at it.

Reply:
I think @iftenney  know this the best. @pruksmhc  and @zphang also know about it.